### PR TITLE
Expand rest of List functions

### PR DESCRIPTION
**Pull request by rocky**

### CODE REVIEW

File: instructions.texi [hunk 1]
```diff
@@ -1754,7 +1754,7 @@ Call @code{aset} with three stack arguments.
 
 @table @strong
 @item Implements:
-@code{S[1] <- (aref S[1] TOS); top--}.
```
Comment (pipcet) [inline instructions.texi:1757]: Should this be `aset`?

File: instructions.texi [hunk 2]
```diff
@@ -1816,48 +1815,209 @@ Constants Vector: [ntchcdr (1550 1551 1552) 2]
...
-Added in Emacs 19.34.
 @node nconc
 @unnumberedsubsubsec @code{nconc} (164)
 @kindex nconc
 Call @code{nconc} with two stack arguments.
 
-Added in Emacs 19.34.
+@table @strong
+@item Implements:
+@code{S[1] <- (nconc S[1] TOS; top--)}.
```
Comment (pipcet) [inline instructions.texi]: You sometimes have the `top--` part in the parens, and sometimes after them. Which one is the official version, so I can fix the others?

**Review by pipcet — APPROVED** [review]
Excellent, just some questions about which way to clean up things when I get around to it

**Comment by rocky** [comment]
Ooops. The intent was to show the Lisp function call in lisp style. Yes, adjusting `top` is using C syntax which isn't Lisp. I find that convenient just because it is shorter. But then the `<-` for assignment also isn't Lisp. 

We don't really document the language we are writing this in, and we could use strictly C syntax, if that would be preferable. What do you think?

**Comment by rocky** [comment]
One other thing. If it's not clear I'm working forward and skipping obsolete (as of emacs 25) instructions. So if you work from the other end, we will be less likely to collide in work. 

After the not obsolete instructions are finished, I do intend to go back an document the obsolete instructions, but that will no doubt means older versions of Emacs so I can check their behavior and run `byte-recalc` on them. (And in the process we'll see what code generation changes on the other examples.

**Comment by pipcet** [comment]
I like the current mix of C and Lisp syntax, and I think the `<-` is understandable enough. I don't think the manual will be useful to those who aren't familiar with both C and Lisp, so I wouldn't worry too much about which one is used, though I definitely prefer Lisp being used for the description of the actual Lisp function calls.

**Comment by rocky** [comment]
Got it.

**Pull request merged**